docs/sources/enterprise/usage-insights/export-logs.md

```diff
@@ -0,0 +1,84 @@
++++
+title = "Export logs of usage insights"
+description = "Export logs of usage insights"
+keywords = ["grafana", "export", "usage-insights", "enterprise"]
+aliases = ["/docs/grafana/latest/enterprise/usage-insights/export-logs.md"]
+weight = 500
++++
+
+# Export logs of usage insights
+
+> **Note:** Available in Grafana Enterprise v7.4+.
+
+By exporting usage logs to Loki, you can directly query them and create dashboards of the information that matters to you most, such as dashboard errors, most active organizations, or your top-10 most-used queries. Currently, usage logs can only be exported to Loki.
```

Comment:
```suggestion
By exporting usage logs to Loki, you can directly query them and create dashboards of the information that matters to you most, such as dashboard errors, most active organizations, or your top-10 most-used queries from within Loki.
```

Reply:
Can we rephrase this to remove the sentiment of insufficiency that comes with the term "only"? Please check that the technical accuracy remains sound wrt my suggestion. Disregard is I have changed the meaning.

Reply:
Made a tweak to remove the Loki mention at the end, since it is mentioned at the beginning. Agree with the suggestion to remove "only" :)